src/components/Hero.astro

```diff
@@ -0,0 +1,77 @@
+<header class="grid">
+  <section>
+    <h1>Regeneracja siłowników hydraulicznych do&nbspkabrioletów</h1>
+    <a href="/"
+      >Zobacz zrealizowane projekty
+      <i class="fas fa-chevron-right"></i>
+    </a>
+  </section>
+  <img class="header-img" src="img/header.svg" alt="" />
+</header>
+
+<style lang="scss">
+  @import '../../styles/sass/base/mixins';
+
+  header {
+    width: 100%;
+    height: 75vh;
+    align-items: center;
+    justify-content: baseline;
+    background-size: cover;
+    background-position-x: center;
+    background-position-y: center;
+
+    // Grid
+    display: grid;
+    grid-template-columns: repeat(2, calc(50% - 24px));
+    grid-gap: 48px;
+
+    @include breakpoint(tablet) {
+      grid-template-columns: 1fr;
+      width: auto;
+      height: auto;
+      padding: 64px 0;
+    }
+
+    section {
+      flex: 1;
+      max-width: 50%;
+
+      @include breakpoint(tablet) {
+        order: 2;
+        max-width: none;
+      }
+
+      h1 {
+        margin: 16px 0;
+      }
+
+      a {
+        display: inline-block;
+        transition: ease-in;
```

Comment:
🤔🤔 do we animate it at all?

Reply:
The content none should hide it, so i believe no need to setup the transitions here

Reply:
I removed the "a transition: easy-in", like you said, I just get it.